packages/remark-wiki-link/test/remarkWikiLink.spec.ts

```diff
@@ -185,8 +189,32 @@ describe("remark-wiki-link", () => {
       });
     });
 
+    test("Alias with accent", () => {
+      const processor = createMarkdownProcessorWithWikiLinkPlugin();
+
+      let ast = processor.parse("[[link|Alias with àcèôíã]]");
+      ast = processor.runSync(ast);
+      expect(select("wikiLink", ast)).not.toEqual(null);
+
+      visit(ast, "wikiLink", (node: Node) => {
+        expect(node.data?.exists).toEqual(false);
+        expect(node.data?.permalink).toEqual("link");
+        expect(node.data?.alias).toEqual("Alias with àcèôíã");
+        expect(node.data?.hName).toEqual("a");
+        expect((node.data?.hProperties as any).className).toEqual(
+          "internal new"
+        );
+        expect((node.data?.hProperties as any).href).toEqual(
+          "link"
+        );
+        expect((node.data?.hChildren as any)[0].value).toEqual(
+          "Alias with àcèôíã"
+        );
+      })
+    })
+
```

Comment:
This test is not needed I think. Aliases are not extracted with regex. Anything between the alias divider and the closing brackets is recognized as a wikilink alias and tokenized as such. Take a look at this file for ref: https://github.com/datopian/portaljs/blob/main/packages/remark-wiki-link/src/lib/syntax.ts